backend/app/Http/Controllers/BrandController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StoreBrandRequest;
+use App\Http\Requests\UpdateBrandRequest;
+use App\Http\Resources\BrandResource;
+use App\Models\Brand;
+use Illuminate\Http\Request;
+
+class BrandController extends Controller
+{
+    public function index()
+    {
+        $brands = Brand::latest()->paginate(10);
+        return BrandResource::collection($brands);
+    }
+
+    public function store(StoreBrandRequest $request)
+    {
+        Brand::create($request->all());
```

Comment:
Never use $request->all() kasi pwede to ma sql inject magpasok ng input na di mo naman expected. use $request->only()

Reply:
okay sir, will do.